samples/system_time/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _System Time Sample App:
+
+System Time Sample App
+###########
+
+Overview
+********
+This sample demostrates that gettimeofday() and settimeofday() works on frdm
+k64f.
+
+Building and Running
+********************
+
+This sample sets the system time and output system time every 5 seconds to the
```

Comment:
How about:

     This sample initially sets the system time using settimeofday() to 
     Sep 14, 2017 20:02:02 GMT (150529000) and then calls gettimeofday()
     every 5 seconds, displaying the seconds value to the console.

     You can build and execute this sample on a frdm k64f board with: